src/ImageSharp/Common/Extensions/Vector4Extensions.cs

```diff
@@ -63,13 +100,47 @@ public static Vector4 Compress(this Vector4 linear)
         /// </summary>
         /// <param name="gamma">The <see cref="Rgba32"/> whose signal to expand.</param>
         /// <returns>The <see cref="Vector4"/>.</returns>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [MethodImpl(InliningOptions.ShortMethod)]
         public static Vector4 Expand(this Vector4 gamma)
         {
             // TODO: Is there a faster way to do this?
             return new Vector4(Expand(gamma.X), Expand(gamma.Y), Expand(gamma.Z), gamma.W);
         }
 
+        /// <summary>
+        /// Bulk variant of <see cref="Compress(System.Numerics.Vector4)"/>
+        /// </summary>
+        /// <param name="vectors">The span of vectors</param>
+        public static void Compress(Span<Vector4> vectors)
```

Comment:
Really interested in the idea of moving these to `PixelOperations`. The optimization opportunity there is fantastic.